src/containers/LoginPageContainer.ts

```diff
@@ -6,11 +6,8 @@ import { KialiAppAction } from '../actions/KialiAppAction';
 import LoginThunkActions from '../actions/LoginThunkActions';
 
 const mapStateToProps = (state: KialiAppState) => ({
-  token: state.authentication.token,
-  username: state.authentication.username,
-  logging: state.authentication.logging,
-  error: state.authentication.error,
-  message: state.authentication.message
+  status: state.authentication.status,
```

Comment:
Please use `reselect` selectors they are memoizable and therefore more efficient and also composable.

https://github.com/reduxjs/reselect

Everything should be using **reselect  selectors** now.

Reply:
Ok, we can resolve this one in a later PR.